platform/android/MapboxGLAndroidSDK/src/main/res/values/arrays.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <array name="mapbox_attribution_names">
-        <item>&#169; Mapbox</item>
-        <item>&#169; OpenStreetMap</item>
-        <item>Improve this map</item>
-        <item>Telemetry Settings</item>
-    </array>
-    <!-- If editing this array update MapView.ATTRIBUTION_INDEX_IMPROVE_THIS_MAP -->
-    <array name="mapbox_attribution_links" formatted="false" translatable="false">
-        <item>https://www.mapbox.com/about/maps/</item>
-        <item>http://www.openstreetmap.org/about/</item>
-        <item>https://www.mapbox.com/map-feedback/#/%1$f/%2$f/%3$d</item>
```

Comment:
The iOS, macOS, and JS implementations automatically [append](https://github.com/mapbox/mapbox-gl-native/blob/62ea1f21858c69f6921c775ba7a3de201f0514d8/platform/darwin/src/MGLAttributionInfo.mm#L129) this hash to the feedback URL. That way, the Map Feedback tool starts out at the right location, which makes their feedback much more actionable. The feedback link has a `class` attribute of `mapbox-improve-map`.